<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QFrame" name="frame" >
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGraphicsView" name="graphicsView" />
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionNew" />
    <addaction name="actionOpen" />
    <addaction name="actionSave" />
    <addaction name="actionSave_As" />
    <addaction name="actionPrint" />
    <addaction name="actionExit" />
   </widget>
   <widget class="QMenu" name="menuInserisci" >
    <property name="title" >
     <string>Inserisci</string>
    </property>
    <addaction name="actionInserisci_Giocatore" />
   </widget>
   <widget class="QMenu" name="menuOpzioni" >
    <property name="title" >
     <string>Opzioni</string>
    </property>
    <addaction name="actionEffetti_sonori" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuInserisci" />
   <addaction name="menuOpzioni" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew" />
   <addaction name="actionAvanti" />
  </widget>
  <action name="actionNew" >
   <property name="icon" >
    <iconset resource="resource.qrc" >:/icons/pictures/filenew.png</iconset>
   </property>
   <property name="text" >
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen" >
   <property name="text" >
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="text" >
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As" >
   <property name="text" >
    <string>Save As</string>
   </property>
  </action>
  <action name="actionPrint" >
   <property name="text" >
    <string>Print</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="text" >
    <string>Exit</string>
   </property>
  </action>
  <action name="actionInserisci_Giocatore" >
   <property name="text" >
    <string>Giocatore</string>
   </property>
  </action>
  <action name="actionEffetti_sonori" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Effetti sonori</string>
   </property>
  </action>
  <action name="actionAvanti" >
   <property name="icon" >
    <iconset resource="resource.qrc" >:/icons/pictures/avanti.png</iconset>
   </property>
   <property name="text" >
    <string>Avanti</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc" />
 </resources>
 <connections/>
</ui>
